src/workerd/api/node/crypto.h

```diff
@@ -6,8 +6,9 @@ namespace workerd::api::node {
 
 class CryptoImpl final: public jsg::Object {
 public:
-
+  bool checkPrimeSync(v8::Local<v8::ArrayBufferView> bufferView, uint32_t num_checks);
```

Comment:
```suggestion
  bool checkPrimeSync(kj::Array<kj::byte> buffer, uint32_t num_checks);
```

Just use the `kj::Array<kj::byte>` type mapping here?